sdk/tests/conformance2/reading/read-pixels-from-fbo-test.html

```diff
@@ -209,11 +226,13 @@
     texInternalFormat: 'RGB565', texFormat: 'RGB', texType: 'UNSIGNED_BYTE',
     readFormat: 'RGBA', readType: 'UNSIGNED_BYTE',
     clearColor: [0.5, 0.7, 0.2, 0], readBackColor: [127, 178, 51, 0], tol: [17, 9, 17, 0],
+    implReadBackColor: [0.5, 0.7, 0.2, 0], implTol: [0.02, 0.02, 0.02, 0.02],
```

Comment:
I don't think this is robust.  It is very likely to break on drivers other than yours.  You assume the impl-dependent format/type is RGB and UNSIGNED_SHORT_5_6_5, thus the implReadBackColor.  But it is not guaranteed.  What if the format/type is RED/UNSIGNED_BYTE?  Then this implReadBackColor and implTol will just be wrong.

The conformance tests should work across GPUs/drivers.